src/createTokenBridgeFetchTokenBridgeContracts.ts

```diff
@@ -0,0 +1,83 @@
+import { Address, PublicClient } from 'viem';
+import { validParentChainId } from './types/ParentChain';
+import { tokenBridgeCreator } from './contracts';
+import { TokenBridgeContracts } from './types/TokenBridgeContracts';
+
+export type CreateTokenBridgeFetchTokenBridgeContractsParams = {
+  inbox: Address;
```

Comment:
It's fine if we use rollup

Reply:
I'm leaning towards not changing this now. In `getTokenBridgeContracts` we get the inbox from the OrbitTokenBridgeCreated event, and then we need to get the bridge address, and finally the rollup address.
Then, here, in createTokenBridgeFetchTokenBridgeContracts, we would need to get the inbox address from the rollup anyway.

This back and forth is what doesn't convince me for now. If we ever see the need to use `rollup` here in the future (in other use cases), it might make sense to refactor it then.